tests/e2e/sendFunds.test.js

```diff
@@ -17,6 +17,11 @@ describe('Send Funds', () => {
       'BTC', 0.02, 0.001);
   }, TimeoutLength);
 
+  it('sends DASH', async () => {
+    await sendFunds(page, 'yV9hgDzzUz1b47RG9XBogTfrdryMhnw1i6', 'yYo9SQDwYGRZFEYtKDBvJ8LGATgn66vaLw',
```

Comment:
is it correct?
What will be the result if count of funds on yV9hgDzzUz1b47RG9XBogTfrdryMhnw1i6 will be 0?

Reply:
The test case will fail